eslint-example/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+		"globals": {"chrome": true},
```

Comment:
can you also add `"browser": true` ? That's the preferred global to use but `chrome` should be supported by the linter too.